WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
I'd get rid of the `competitor_limit_flag` column.

This function can be implemented with `competitor_limit.present?`.